.github/workflows/release-typescript-sdk.yaml

```diff
@@ -0,0 +1,28 @@
+name: TypeScript SDK Release
+on:
+  workflow_call:
+    secrets:
+      NPM_TOKEN:
+        description: 'NPM token'
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - name: Install pre-requisites
+        uses: ./.github/actions/setup
+       
+      - name: Publish to npm
+        working-directory: sdk/typescript
+        shell: bash
+        run: |
+          set -e
+          ../../gradlew build
+          npm publish
```

Comment:
We should make sure the name in `package.json` matches the LF Decentralized Trust org. I think it should be `@lfdecentralizedtrust-labs/paladin-sdk` based on Discord conversation.